src/lib/components/program/sprint/Schedule.svelte

```diff
@@ -1,122 +1,202 @@
 <script>
-    import Heading from '$lib/components/Heading.svelte'
-    import { prettyDate, longDate } from '$lib/utils/date.js'
+	import Heading from '$lib/components/Heading.svelte';
+	import { prettyDate, longDate } from '$lib/utils/date.js';
 
-    export let weekPlans
+	export let weekPlans;
+
+	const checkKeywords = (content) => {
+		const strings = [
+			'sprintplanning',
+			'kickoff',
+			'workshop',
+			'opdrachtgever',
+			'code review',
+			'retrospect ',
+			'wrap-up',
+			'review'
+		];
+		let newString = '';
+		if (content) {
+			const splitted = content.split('<p>');
+			splitted.forEach((paragraph) => {
+				let toReplace = strings.filter((s) => {
+					if (paragraph.toLowerCase().includes(s)) {
+						return s.replace('we ♥ web', 'we love web', s).replaceAll(' ', '-', s);
```

Comment:
Oja, deze kreeg ik niet aan de praat, ik heb wel 'we ♥ web' als string verwijderd maar hij staat hier nog. Als het jullie lukt hoor ik het graag, anders haal ik het nog weg

Reply:
Very nice Cyd! Ik vind de mobiele versie nu veel beter bruikbaar. Ik ga het pull request mergen. Bedankt voor al je werk!